content/posts/aws-powers-read-the-docs.md

```diff
@@ -0,0 +1,132 @@
+image:
+image_credit:
+
+Read the Docs has been building and hosting documentation for open-source projects since 2010.
+We’re proud to be a part of the open source ecosystem,
+and today we wanted to take a moment to thank one of our biggest supporters: **Amazon Web Services**.
+
+We have been a member of the [AWS Open Source Credits](https://aws.amazon.com/blogs/opensource/aws-promotional-credits-open-source-projects/) program since 2021.
+In that time,
+they have provided us with **almost $500,000 in credits**,
+which covers all of our server-related costs each year for the Community platform.
+
+This post will cover a bit more about how we use AWS,
+the infrastructure we run on it,
+and how it helps us support the open source projects that rely on Read the Docs.
+
+## Our scale
+
+In the past 30 days, our Community hosting platform has:
+
+* Supported over **100,000 open source projects** with documentation hosting
+* Run over **350,000 documentation builds**
+* Handled over **250M documentation pageviews**
+* Served over **2.2 billion web requests**
+* Transferred over **45TB of data**
+
+Since our creation in 2010,
+we have been a cornerstone of the Python open source ecosystem,
+and now host documentation for many other languages and frameworks as well.
+
+## Our infrastructure
+
+Read the Docs has a pretty simple architecture,
+because we have a small team of 5 people running a very large web service.
+The major AWS services that we rely on are:
+
+* **EC2**: We run our web servers, web workers, and build servers on EC2 autoscaling groups.
+  This allows us to scale up and down as needed, leading to builders always being available when projects need them.
+* **S3**: We store all of our documentation artifacts in S3.
+  This includes the HTML files, PDFs, and other outputs generated by our build system.
+* **RDS**: Our primary and replica Postgres databases are hosted on RDS.
+  This allows us to scale our database easily, and we rarely have to think about it.
+
+We use a number of other AWS services as well, including Cloudwatch for monitoring, Route 53 for DNS, and many more.
+
+## A few of the projects we host
+
+Here are just a few of the big projects whose docs we build and host:
+
+- [**Godot Engine**](https://docs.godotengine.org/) – the open-source 2D/3D game engine behind thousands of indie titles and student projects.
+- [**Jupyter**](https://docs.jupyter.org/en/latest/) – the open source project that powers Jupyter Notebooks, used by millions of data scientists and educators worldwide.
+- [**Flask**](https://flask.palletsprojects.com/) – the Python web framework powering everything from hobby APIs to production workloads at scale.
+- [**Python Developers Guide**](https://devguide.python.org/) - the official guide for developing Python itself, maintained by the core developers.
+- Tens of thousands of other open-source libraries. Scroll almost any ``requirements.txt`` and you’ll see at least one project whose docs we host.
```

Comment:
Requirement files feels rather specific and out of date, they are Python only and aren't commonly used by Python packages or modern repositories.

```suggestion
- Tens of thousands of other open-source libraries. Browse the documentation of the libraries, software, and services you use and it won't take long to find a project whose docs we host.
```